src/lib.rs

```diff
@@ -374,6 +374,30 @@ impl Vhal {
         self.set_property(VehicleProperty::AP_POWER_STATE_REQ, value, 0, None)
     }
 
+    pub fn set_battery_level(&self, speed: f32) -> Result<()> {
```

Comment:
Oops, I just saw after merging that the parameter name was speed. Could you please post a quick fix to change it?

Reply:
https://github.com/aesteve-rh/vhal_emulator/pull/8